sync-core/src/main/java/com/cloudant/sync/replication/Replication.java

```diff
@@ -29,9 +30,8 @@
 /**
  * <p>Abstract class which provides configuration for a replication.</p>
  *
- * <p>This class is abstract. Concrete classes
- * {@link com.cloudant.sync.replication.PullConfiguration}
- * and {@link com.cloudant.sync.replication.PushReplication} are used to
+ * <p>This class is abstract. Concrete class {@link com.cloudant.sync.replication.PushReplication}
```

Comment:
This doesn't make sense any more - the link is for PushReplication but the following text mentions push and pull